base-image/start

```diff
@@ -3,6 +3,11 @@
 # ==== ONLY EDIT WITHIN THIS BLOCK =====
 
 export PANGEO_ENV="base"
+if [[ -z "${PANGEO_SCRATCH_PROTOCOL}" ]]; then
+    :
+else
+    export PANGEO_SCRATCH="${PANGEO_SCRATCH_PROTOCOL}://pangeo-scratch/${JUPYTERHUB_USERNAME}/"
```

Comment:
I'm thinking it might be better to absorb the bucket name into the variable provided by the hub, rather than hard coded. So something like

```
PANGEO_SCRATCH="${PANGEO_SCRATCH_PREFIX}/${JUPYTERHUB_USERNAME}/"
```